PSKoans/Koans/Constructs and Patterns/AboutRegularExpressions.Koans.ps1

```diff
@@ -0,0 +1,584 @@
+            quantifier by simply adding a ?.
+            #>
+
+            $lazyPattern = 'p+?'
+            $lazyMatch = [regex]::Matches('pineapple', $lazyPattern)[0].Value
+            __ | Should -Be $lazyMatch.Length
+        }
+
+        It 'is a hungry bugger' {
+            $greedyPattern = 'a+'
+            $greedyMatch = [regex]::Matches('aardvark', $greedyPattern)[0].Value
+            __ | Should -Be $greedyMatch.Length
+        }
+
+        It 'needs to decide which is lazy and which is greedy' {
+            $greedy = 'o+'
+            $lazy = 'o+?'
+            @([regex]::Matches('helloooooo', $____).Value)[0] | Should -Be 'o'
+            @([regex]::Matches('helloooooo', $____).Value)[0] | Should -Be 'oooooo'
+        }
+    }
+}
+
+Describe 'Special Symbols' {
+    <#
+        In regex, there are far more symbols with unique meanings than just quantifiers. What comes
+        next is not a definitive guide to EVERY SINGLE ONE, but rather an introduction to some of
+        the most common and useful special symbols in regex.
+        
+        This is ^\where things st\Art getting \weir\D$
+    #>
+    
+
+    Context '. (period)' {
+
+        It 'matches any character' {
+            # The . (period) matches literally any character
+    
+            $____ | Should -Be 'Lazy Sunday mornings' -match 'S.nday'
+        }
+
+        It 'starts looking kind of weird now' {    
+            '____' -match 'invi.a.ion' | Should -BeTrue
+        }
+    }
+
+    Context '\n' {
+
+        It 'matches new lines' {
+            <#
+                The \n symbol is the first regex symbol you've encountered that isn't just a single
+                character. Normally if you saw a lowercase "n" in a regex, it would just mean
+                "literally the letter n". When it's preceded by a backslash, however, that "n" takes
+                on a special meaning. In this case, \n matches new lines. Actually, it matches the
+                encoded character that indicates a new line.
+    
+                There are many more regex symbols that are single letters preceded by a backslash.
+                In fact, the backslash is probably the single most important character in all of
+                regex.
+            #>
+    
+            $multiLine = @"
+They might look
+similar and look
+soft, but you should
+never confuse a
+wild cougar for
+a domestic cat
+"@
+            $catPattern = '____\n'
+            $multiLine -match $catPattern | Should -BeTrue
+        }
+    }
+
+    Context '\d and \D' {
+
+        It 'matches digits' {
+            <#
+                The \d symbol matches digits (the numbers 0 - 9 inclusive), while the symbol \D
+                matches anything OTHER THAN a digit. Yes, regular expressions are case sensitive.
+                This "lower case for affirmative, upper case for negative" pattern is common among
+                regex symbols as you'll discover as you carry on.
+            #>
```

Comment:
Given this refers to the context block as a whole, this comment should probably just be directly under the Context header. I think you may have a small handful of those; would be a good idea to check over those and make sure that the comments are consistent and in the right places. 🙂